src/test/java/io/github/sskorol/entities/TestData.java

```diff
@@ -0,0 +1,44 @@
+package io.github.sskorol.entities;
+
+import com.creditdatamw.zerocell.annotation.Column;
+import io.github.sskorol.converters.*;
+import io.github.sskorol.data.Sheet;
+import io.github.sskorol.data.Source;
+import lombok.Getter;
+import lombok.ToString;
+
+import java.time.LocalDate;
+import java.util.List;
+
+@ToString
```

Comment:
You can use `@Data` annotation which covers both `@ToString` and `@Getter`

Reply:
@Data also adds setter methods, hashcode implementation. Could you please confirm so that I can change to @Data

Reply:
Yes, it aggregates all common annotations.